runtime/src/main/resources/packaged/configuration.yml

```diff
@@ -79,5 +80,9 @@ eventScheduler:
     batchSize: 50
     intervalms: 10000
 
+eventProxyForMigration:
+  enabled: "yes"
+  endpoint: "http://10.85.42.144"
```

Comment:
avoid committing IPs

Reply:
was needed for the new tests to pass, will remove now.